Makefile

```diff
@@ -116,17 +116,25 @@ clean:
 	cd lib/elixir && $(REBAR) clean
 	rm -rf ebin
 	rm -rf lib/*/ebin
-	rm -rf lib/elixir/test/ebin
-	rm -rf lib/*/tmp
-	rm -rf lib/mix/test/fixtures/git_repo
-	rm -rf lib/mix/test/fixtures/deps_on_git_repo
-	rm -rf lib/mix/test/fixtures/git_rebar
 	rm -rf lib/elixir/src/elixir.app.src
-	$(MAKE) clean_man
+	rm -f Docs-v*.zip
+	rm -f Precompiled-v*.zip
+	$(Q) $(MAKE) clean_residual_files
 
 clean_exbeam:
 	$(Q) rm -f lib/*/ebin/Elixir.*.beam
 
+clean_residual_files:
+	rm -rf lib/*/_build/
+	rm -rf lib/*/tmp/
+	rm -rf lib/elixir/test/ebin/
+	rm -rf lib/mix/test/fixtures/deps_on_git_repo/
+	rm -rf lib/mix/test/fixtures/git_rebar/
+	rm -rf lib/mix/test/fixtures/git_repo/
+	rm -rf lib/mix/test/fixtures/git_sparse_repo/
+	rm -f erl_crash.dump
```

Comment:
@josevalim not sure if you didn't want to remote erl_crash.dump in subfolder, or that you didn't want to remove erl_crash.dump at all